### ISSUE TITLE
should we rename 'watch' to be 'deploy-watch'?

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8-maven-plugin#368]
based on this #364 - I wonder if watch is too vague? A deploy prefix might be more clear its a deploy-and-watch kinda thing?

**Comment by davsclaus** [comment]
Yeah its sad that mvn dont have parameters with just `--watch` or someting.

**Comment by rhuss** [comment]
we could add a `-Dfabric8.watch` as replacement for a parameter (the "Maven way")
